src/audit/mod.rs

```diff
@@ -0,0 +1,25 @@
+use std::{env, fs};
+use uuid::Uuid;
+
+/// Internal logging function used to log errors.
+/// Can be disabled by setting environment variable ACCEPTEVM to 0
+pub fn log_sync(data: &str) {
+    match env::var("ACCEPTEVM_LOGS") {
```

Comment:
usually everyone uses RUST_LOG env
where the crate/module can be specified
https://rust-lang-nursery.github.io/rust-cookbook/development_tools/debugging/config_log.html